.mailmap

```diff
@@ -140,6 +157,7 @@ Beta Ziliani <beta@mpi-sws.org>                    Beta Ziliani <bziliani@famaf.
 Beta Ziliani <beta@mpi-sws.org>                    beta <beta@mpi-sws.org>
 Théo Zimmermann <theo.zimmermann@univ-paris-diderot.fr> Theo Zimmermann <theo.zimmermann@ens.fr>
 Théo Zimmermann <theo.zimmermann@univ-paris-diderot.fr> Théo Zimmermann <theo.zimmi@gmail.com>
+Théo Zimmermann <theo.zimmermann@univ-paris-diderot.fr> Théo Zimmermann <theo.zimmermann@inria.fr>
```

Comment:
The Inria address is the new one. Can you use this one instead?

Reply:
Done